### ISSUE TITLE
Icon for Apache Directory Studio

### ISSUE THREAD

**Issue by mikestephens** [numixproject/numix-core#2544]
There is no icon for Apache Directory Studio

apache_directory_studio.desktop 

```
[Desktop Entry]
Encoding=UTF-8
Version=1.0
Type=Application
Name=LDAP - Apache Directory Studio 
Icon=apache_directory_studio.png
Path=/opt/ApacheDirectoryStudio
Exec=/usr/bin/java -jar /opt/ApacheDirectoryStudio//plugins/org.eclipse.equinox.launcher_1.3.0.v20140415-2008.jar -os linux -ws gtk -arch x86_64 -showsplash -launcher /opt/ApacheDirectoryStudio/ApacheDirectoryStudio -name ApacheDirectoryStudio --launcher.library /opt/ApacheDirectoryStudio//plugins/org.eclipse.equinox.launcher.gtk.linux.x86_64_1.1.200.v20150204-1316/eclipse_1607.so -startup /opt/ApacheDirectoryStudio//plugins/org.eclipse.equinox.launcher_1.3.0.v20140415-2008.jar --launcher.overrideVmargs -exitdata 5c8007 /studio-rcp/resources/icons/linux/studio.xpm -vm /usr/bin/java -vmargs -jar /opt/ApacheDirectoryStudio//plugins/org.eclipse.equinox.launcher_1.3.0.v20140415-2008.jar
StartupNotify=false
StartupWMClass=Apache Directory Studio
OnlyShowIn=Unity;
X-UnityGenerated=true

```

![](http://i.imgur.com/aLC6RfB.png)

here is a higher res tree:

![](http://i.imgur.com/Tyfr9zA.jpg)

I am fairly sure it is a hardcoded icon but I'd love to have one to manually set it.

Thanks for all your work!
##

**Comment by Foggalong** [comment]
We still need the information setout in the readme, even for hardcoded icons. This includes the file name of the .desktop file :)

**Comment by mikestephens** [comment]
whoops! sorry about that. I edited the original post. 

I am fairly sure you just need the icon name from the .desktop file but I thought why not give you the whole thing :p

**Comment by Foggalong** [comment]
Thanks :D

**Comment by really-need-an-api-key** [comment]
Just browsing through old issues, I think this was solved in b3f6ad16b44ce6998eb14de101e4ed605a4394d4.
